examples/ads.amp.html

```diff
@@ -317,6 +318,14 @@ <h2 id="adverline">Adverline</h2>
     <div fallback></div>
   </amp-ad>
 
+  <amp-ad width=300 height=250
+     type="adverticum"
```

Comment:
I am not able to see a rendered ad example. Got Error: `timeout wait for ad response`. Please check and update the example.